v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/dlq/BigQueryDeadLetterQueue.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.kafka.dlq;
+
+import com.google.cloud.teleport.metadata.TemplateParameter;
+import org.apache.beam.sdk.options.Default;
+
+public interface BigQueryDeadLetterQueue {
```

Comment:
By convention, all pipeline options interfaces end with "Options", i.e. this one should be "BigQueryDeadLetterQueueOptions".

Reply:
Done